thorclient/http/client.go

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package http
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/vechain/thor/v2/api/accounts"
+	"github.com/vechain/thor/v2/api/blocks"
+	"github.com/vechain/thor/v2/api/events"
+	"github.com/vechain/thor/v2/api/node"
+	"github.com/vechain/thor/v2/api/transactions"
+	"github.com/vechain/thor/v2/api/transfers"
+	"github.com/vechain/thor/v2/thor"
+	"github.com/vechain/thor/v2/thorclient/common"
+)
+
+type Client struct {
+	url string
+	c   *http.Client
+}
+
+func NewClient(url string) *Client {
+	return &Client{
+		url: url,
+		c:   &http.Client{},
+	}
+}
+
+func (c *Client) GetTransactionReceipt(txID *thor.Bytes32) (*transactions.Receipt, error) {
```

Comment:
Nothing wrong with the name, but I would have a preference to remove the `Get`s from the function names. Seems we have a lot of functions here prefixed with `Get`